personio-personnel-data-api-oa3.yaml

```diff
@@ -3828,6 +3970,64 @@ components:
         - end_date
         - half_day_start
         - half_day_end
+    CreateAbsencePeriodRequest:
+      title: Create Absence periods **tracked in hours**
+      type: object
+      properties:
+        employee_id:
+          type: integer
+          description: Employee identifier
+          example: 10735
+        absence_type_id:
+          type: string
+          format: uuid
+          description: Absence type identifier
+          example: 9bba303f-0fbc-4514-9958-0befa21923fb
+        start_date:
+          type: string
+          format: date
+          description: "Absence start date. Format: yyyy-mm-dd"
+          example: '2020-01-31'
+        end_date:
+          type: string
+          format: date
+          description: "Absence end date. Format: yyyy-mm-dd"
+          example: '2020-01-31'
+        full_day:
+          type: boolean
+          example: false
+          description: Whether it's a full day absence or not, only required if the start_date and end_date are the same (one day absence at most)
+        start_time:
+          type: string
+          format: date
+          description: "Absence start time. Format: HH:mm (hours and minutes) and only required if the start_date and end_date are the same, and full_day is false (partial-day absence)"
```

Comment:
should we mention in which time zone the start_time, end_time and full_day are being applied?

Reply:
In all our endpoints we usually receive UTC time from the client, but why not mentioning this info here as well, the more descriptive the better :) changing it 👍